documentation/sphinx/source/release-notes.rst

```diff
@@ -26,6 +26,8 @@ Status
 Bindings
 --------
 
+* C: Added ``fdb_transaction_get_approximate_size`` to get the approximated transaction size before commit. `(PR #1756) <https://github.com/apple/foundationdb/pull/1756>`_.
```

Comment:
This should be a more generic release note since we added an API for this in all of the bindings.